### DISCUSSION TITLE
编辑器按住中键移动出现bug

### DISCUSSION THREAD

**Discussion by Gala-TA** [galacean/discussion#92] [Editor]
https://github.com/galacean/engine/assets/35087284/edc1f2a7-ffcb-4d81-8367-40fa6b2ffd12

**Comment by JujieX** [comment]
补充说明：
左边的Hierarchy的面板没了，右边出现一个空白的面板
![image](https://github.com/galacean/engine/assets/67930474/f69260ba-c711-45d3-88df-a0e43a3a9669)
